src/components/composer/style.js

```diff
@@ -1,17 +1,56 @@
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 import theme from 'shared/theme';
 import { hexa, Shadow, FlexRow, FlexCol, zIndex } from '../globals';
 
+export const ComposerSlider = styled.div`
+  display: ${props => (props.isSlider ? 'none' : 'block')};
+  ${props =>
+    props.isSlider &&
+    props.isOpen &&
+    css`
+      display: block;
+      position: fixed;
+      width: 50%;
+      top: 0;
+      right: 0;
+      bottom: 0;
+      z-index: ${zIndex.composer};
+    `}
+`;
+
+export const Overlay = styled.div`
+  ${props =>
+    props.isOpen
+      ? `
+      position: fixed;
+      top: 48px;
+      left: 0;
+      right: 0;
+      bottom: 0;
+      width: 100%;
+      height: 100%;
+      z-index: 3000;
+      background: #000;
+      pointer-events: auto;
+      opacity: 0.4;
+    `
+      : `
+      opacity: 0;
+      pointer-events: none;
+
+    `};
+`;
+
 export const Container = styled(FlexCol)`
   background-color: ${theme.bg.default};
   display: grid;
-  grid-template-rows: 60px 1fr 64px;
+  grid-template-rows: 50px 1fr 64px;
   grid-template-columns: 100%;
   grid-template-areas: 'header' 'body' 'footer';
   align-self: stretch;
   flex: auto;
   overflow: hidden;
-  height: calc(100vh - 48px);
+  height: 100vh;
```

Comment:
This should not change, otherwise the thread composer overflows on `/new/thread`! (48px is the navbar height)

```suggestion
  height: calc(100vh - 48px);
```

Reply:
oh I missed that, probably an idea to only add those styles with the isSlider props, similar to other styles